sympa.spec

```diff
@@ -207,19 +207,6 @@ Requires:    impallari-raleway-fonts
 #            http://zurb.com/playground/uploads/upload/upload/288/foundation-icons.zip
 
 # Bundled javascript libs
-# foundation
-%if %{unbundle_foundation}
-Requires:    js-foundation5
-%else
-Provides:    bundled(js-foundation) = 5.5.3
-# Bundled in bundled js-foundation
-Provides:    bundled(js-fastclick)
-Provides:    bundled(js-jquery-cookie) = 1.4.1
-Provides:    bundled(js-jquery-requestAnimationFrame)
-Provides:    bundled(js-modernizr) = 2.8.3
-Provides:    bundled(js-placeholder) = 2.0.9
-Provides:    bundled(js-sizzle) = 2.2.0
-%endif
```

Comment:
This part shouldn't be removed but updated for foundation 6:
```
%if %{unbundle_foundation}
Requires:    js-foundation6
%else
Provides:    bundled(js-foundation) = 6.4.2
# Bundled in bundled js-foundation
Provides:    bundled(js-what-input) = 4.2.0
%endif
```

Reply:
I repushed commits according to your suggestion.